index.js

```diff
@@ -108,10 +108,18 @@ module.exports = (helpText, options) => {
 	delete argv._;
 
 	const flags = camelcaseKeys(argv, {exclude: ['--', /^\w$/]});
+	const unnormalizedFlags = camelcaseKeys(argv, {exclude: ['--', /^\w$/]});
+
+	if (options.flags !== undefined) {
+		for (const flag of Object.entries(options.flags)) {
```

Comment:
`for (const flag of Object.entries(options.flags)) {` => `for (const flagValue of Object.values(options.flags)) {`